src/components/TodoContainer.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable max-len, no-param-reassign, react/destructuring-assignment, react/no-access-state-in-setstate, react/state-in-constructor */
+
```

Comment:
Please make the same changes as stated above. You can use this [link as a guide](https://github.com/microverseinc/linters-config) provided by Microverse  👍